.github/workflows/template.bicep.test.yml

```diff
@@ -72,10 +73,15 @@ jobs:
           WORKSPACE_PATH: ${{ github.workspace }}
           RUN_ID: ${{ github.run_id }}
         run: |
-          export ARM_TENANT_ID=${{ steps.LoadSecrets.outputs.tenantId }}
-          export ARM_SUBSCRIPTION_ID=${{ steps.LoadSecrets.outputs.subscriptionId }}
-          export ARM_CLIENT_ID=${{ steps.LoadSecrets.outputs.clientId }}
-          export ARM_CLIENT_SECRET=${{ steps.LoadSecrets.outputs.clientSecret }}
+          export ARM_TENANT_ID="${{ steps.LoadSecrets.outputs.tenantId }}"
+          export ARM_SUBSCRIPTION_ID="${{ steps.LoadSecrets.outputs.subscriptionId }}"
+          export ARM_CLIENT_ID="${{ steps.LoadSecrets.outputs.clientId }}"
+          export ARM_CLIENT_SECRET="${{ steps.LoadSecrets.outputs.clientSecret }}"
+
+          export AZ_TENANT_ID="${{ steps.LoadSecrets.outputs.tenantId }}"
+          export AZ_SUBSCRIPTION_ID="${{ steps.LoadSecrets.outputs.subscriptionId }}"
+          export AZ_APPLICATION_ID="${{ steps.LoadSecrets.outputs.clientId }}"
+          export AZ_ENCRYPTED_PASSWORD="$(pwsh -c "\"${{ steps.LoadSecrets.outputs.clientSecret }}\" | ConvertTo-SecureString -AsPlainText -Force | ConvertFrom-SecureString")"
```

Comment:
Just to be extra safe, I'd also mask the encrypted password from the pipeline output

Reply:
We are only setting environment variables, and they do not appear in the terminal logs. Therefore, there's no need to mask them as they are not logged in to the session.